apps/mobile-wallet/src/features/localization/useLocalization.ts

```diff
@@ -18,6 +18,7 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import 'dayjs/locale/fr'
 import 'dayjs/locale/el'
+import 'dayjs/locale/vi'
```

Comment:
Should Greek and German also be here?

Reply:
Greek is there, the second row (`el`). We don't have full German translations yet, it's very incomplete, so I'd rather wait until we have full coverage before enabling it.